src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -247,8 +247,10 @@ namespace {
 #define BLE_CONFIG_MAX_CE_LENGTH (0xFFFF) // Leave to max value
 
 #define BLE_DEFAULT_TIMER_PERIOD_MS (1)
+#define BLE_SEND_INDICATION_TIMER_PERIOD_MS (400)
```

Comment:
shouldn't these be in the header file?

Reply:
`#define BLE_TIMEOUT_MS 400` does this serve a different purpose than the above time period?

Reply:
Yes this for indication Timeout